test/testenv/testenv_test.go

```diff
@@ -0,0 +1,127 @@
+package testenv
+
+import (
+	"flag"
+	"io"
+	"log"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/pborman/uuid"
+	"golang.org/x/net/context"
+	mixerapi "istio.io/api/mixer/v1"
+	"istio.io/mixer/adapter/denier"
+	"istio.io/mixer/pkg/attribute"
+	"istio.io/mixer/pkg/handler"
+)
+
+func closeHelper(c io.Closer) {
```

Comment:
I think all the helper methods in this file should be in the testenv.go.. because every adapter test will be using them. It will be super slick if all they have to type is the content of your sample 'TestDenierAdapter'

Reply:
agree

Reply:
I am hoping the help methods of copying files to creating config store aren't not needed in the future by using memory based config store. closerHelper could be generally helpful, so it could be move to some common package. For now, I would prefer keep the test framework compact.